app/views/granules/prepare_script.html.erb

```diff
@@ -126,7 +131,7 @@ prompt_credentials
 }
 
 fetch_urls <<'EDSCEOF'
-  </pre>
```

Comment:
The two space indentation here is what was causing the first link in the list in the script to also be indented - we don't want that.  The problem is 'pre' keeps things literal, so it also adds the spaces to the script it creates.